db/migrate/20201222070116_add_index_to_search_documents_content.rb

```diff
@@ -0,0 +1,11 @@
+class AddIndexToSearchDocumentsContent < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def change
+    add_index :pg_search_documents,
+              %[to_tsvector('simple', coalesce("pg_search_documents"."content"::text, ''))],
```

Comment:
Note: since we support multiple languages (at least on DEV) we can't use a language-specific dictionary here. This means normalization steps like stemming and stop word removal will be skipped:

```sql
PracticalDeveloper_development# SELECT to_tsvector('simple', 'The men ran, the woman runs');
┌──────────────────────────────────────────────┐
│                 to_tsvector                  │
╞══════════════════════════════════════════════╡
│ 'men':2 'ran':3 'runs':6 'the':1,4 'woman':5 │
└──────────────────────────────────────────────┘
(1 row)

PracticalDeveloper_development# SELECT to_tsvector('english', 'The men ran, the woman runs');
┌───────────────────────────────────┐
│            to_tsvector            │
╞═══════════════════════════════════╡
│ 'men':2 'ran':3 'run':6 'woman':5 │
└───────────────────────────────────┘
(1 row)
```

Reply:
Makes sense, we also don't know which languages other Forems use or will use